### ISSUE TITLE
sequential single quote characters get stripped from note.

### ISSUE THREAD

**Issue by charlierlee** [jaap-karssenberg/zim-desktop-wiki#539]
If I write something like 'select * from table where name = ''blah'' '

the ''blah'' gets changed to blah after closing and reopening.

**Comment by jaap-karssenberg** [comment]
Yes, this is a feature of using wiki syntax.  If you want to quote
statements like this, please use the verbatim mark up to protect them.

-- Jaap

**Comment by charlierlee** [comment]
It just means I can't use this app to quickly take notes on some code that I wrote.
